src/main/java/com/ita/if103java/ims/mapper/jdbc/EventRowMapper.java

```diff
@@ -10,10 +10,9 @@
 import java.time.LocalDateTime;
 import java.time.ZoneId;
 import java.time.ZonedDateTime;
-import java.util.function.Consumer;
 
 @Component
-public class EventRowMapper implements RowMapper<Event> {
+public class EventRowMapper extends AbstractRowMapper implements RowMapper<Event> {
```

Comment:
I guess, you wanna rid of `setValueOrNull` method. Probably you could do something like that 
Move it to `RowMapperUtils`:
```java
class RowMapperUtils {
    public static <T> void setValueOrNull(Consumer<T> consumer, T value, ResultSet rs) throws SQLException {
        consumer.accept(rs.wasNull() ? null : value);
    }
}
```
And use in the next way:
```java
import static com.....RowMapperUtils.setValueOrNull
.....

   @Override
    public Event mapRow(ResultSet resultSet, int i) throws SQLException {
        Event event = new Event();
        setValueOrNull(event::setTransactionId, resultSet.getLong("transaction_id"), resultSet);
        return event;
    }
```

Reply:
Yeah, you won an automobile )

Reply:
Thanks)